std/math.d

```diff
@@ -1592,21 +1585,10 @@ real exp(real x) @trusted pure nothrow @nogc
         enum real C2 = 1.428606820309417232121458176568075500134E-6L;
 
         // Overflow and Underflow limits.
-        static if (real.mant_dig == 64) // 80-bit reals
```

Comment:
To re-cap what I've already raised - for the *third* time.

This line is not using the ``RealFormat`` enum, it should be ``F.realFormat == RealFormat.ieeeExtended``, likewise the static if below too (``F.realFormat == RealFormat.ieeeDouble``)